CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+set(CMAKE_AUTOUIC ON)
+set(CMAKE_AUTOMOC ON)
+set(CMAKE_AUTORCC ON)
+
+cmake_minimum_required(VERSION 3.5)
+project(solar-car-dashboard)
+
+# Find the required Qt packages
+find_package(QT NAMES Qt5 Qt6 COMPONENTS Core Quick)
```

Comment:
Why are we adding qt5 here? Just wondering if something was added/changed to make it necessary

Reply:
The linux apt package of qt6 does not currently have quick module I believe, thus it's looking for both versions

Reply:
Sg